cmd/lncli/invoicesrpc_active.go

```diff
@@ -252,11 +252,7 @@ func addHoldInvoice(ctx *cli.Context) error {
 		return err
 	}
 
-	printJSON(struct {
-		PayReq string `json:"pay_req"`
-	}{
-		PayReq: resp.PaymentRequest,
-	})
+	printRespJSON(resp)
```

Comment:
Btw, this update changes one of the json response items from "pay_req" to "payment_request", as defined in the proto file. For this and future updates, is this a big deal?

Reply:
I think that's okay. Hopefully scripts use the gRPC interface directly and don't parse the output of `lncli`.